controllers/storagecluster/reconcile.go

```diff
@@ -647,3 +677,24 @@ func validateArbiterSpec(sc *ocsv1.StorageCluster, reqLogger logr.Logger) error
 	}
 	return nil
 }
+
+func updateRookConfig(defaultRookConfigData string, val string) (string, error) {
```

Comment:
I meant we could change the method to accept a section and key to change too so that it can change keys other than `public_network` and in sections other than `global`

Reply:
done